warcry_proj/warcry_proj/views/default.py

```diff
@@ -0,0 +1,139 @@
+    if user_id == '__first__':
+        user_id = UserSchema().dumps(DBSession.query(User).get(1), cls=DateTimeEncoder)
+        user_id = UserSchema().dumps(DBSession.query(User).get(1)).id
+
+    user = UserSchema().dumps(DBSession.query(User).get(user_id), cls=DateTimeEncoder)
+    if not user:
+        msg = "The User with id '{}' was not found.".format(user_id)
+        return Response(status=404, json_body={'error': msg})
+
+    return user
+
+#Create / Insert a User
+@view_config(route_name='create_new_user', request_method='POST', renderer='json')
+def create_user(request):
+     data = request.json_body
+     schema = UserSchema()
+     json_dict = schema.load(data).data
+     entered_name = json_dict["name"]
+     #Super Hero
+     superHero = json_dict["super_hero"] if json_dict.get("super_hero") else False
+     #Created At
+     createdAt = json_dict["created_at"] if json_dict.get("created_at") else datetime.now()
+     #super_hero = int(json_dict["super_hero"])
+     #createdAt = json_dict["created_at"]
+     #Get Record With Max Record ID in Database
+     #current_user_from_db = DBSession().query(User).order_by(User.id.desc()).first()
+     #user = User(name='Bruce Wayne1', super_hero=True, created_at=datetime.now())
+     #user = User(name=entered_name, super_hero=superHero, created_at=createdAt)
+     #DBSession.add(user)
+     # Flush to get the post.id from the database
+     #DBSession.flush()
+     #DBSession.commit()
+     with transaction.manager:
+        user = User(name=entered_name, super_hero=superHero, created_at=createdAt)
+        DBSession.add(user)
+        DBSession.flush()
+
+     return "User Created Successfully"
+
+#Update User
+@view_config(route_name='update_user', request_method='PUT', renderer='json')
+def update_user(request):
+    user_id = request.matchdict.get('user_id')
+    data = request.json_body #request.POST
+    schema = UserSchema()
+    json_dict = schema.load(data).data
+    #Get Current Record in Database
+    current_user_from_db = DBSession().query(User).filter_by(id=user_id).first()
+    #Name
+    new_name = json_dict["name"] if json_dict.get("name") else current_user_from_db.name
+    #Super Hero
+    superHero = json_dict["super_hero"] if json_dict.get("super_hero") else current_user_from_db.super_hero
+    #Created At
+    createdAt = json_dict["created_at"] if json_dict.get("created_at") else current_user_from_db.created_at
+
+    with transaction.manager:  
+       DBSession.query(User).filter_by(id=user_id).update({"name": new_name,"super_hero": superHero, "created_at":createdAt})
+    return "User Updated Successfully"
+
+#DELETE user
+@view_config(route_name='delete_user', request_method='DELETE', renderer='json')
+def delete_user(request):
+    user_id = request.matchdict.get('user_id')
+    user = DBSession.query(User).filter_by(id=user_id).first()
+    if not user:
+        msg = "The User with id '{}' was not found.".format(user_id)
```

Comment:
As with above, by returning a 404, you're already saying "not found", you don't need to add the user message of not found. You get that for free as part of it being a REST API.